package/src/custom/gradebook_fake/RecordManager.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Sugarcrm\Sugarcrm\custom\gradebook_fake;
+
+/**
+ * Class RecordManager
+ * @package GradebookFake
+ * This class manages the records for GradebookFake.
+ * No records are actually saved!
+ */
+class RecordManager
+{
+    /**
+     * This function pretends to create a student record in GradebookFake
+     * @param $email
+     * @param $firstName
+     * @param $lastName
+     * @return bool true if the record was created in GradebookFake
+     * @throws \Exception if the email address forceerror@example.com is used
+     */
+    public function createStudentRecord($email, $firstName, $lastName)
+    {
+        if ($email === 'forceerror@example.com'){
+            throw new \Exception("An error was forced because the email address forceerror@example.com was used.");
+        }
+        return true;
```

Comment:
Realistically, this would return some sort of external ID for the newly created Student record that we'd want to store in Sugar. Email addresses and names don't make good IDs since they can change. I think we should return a UUID. As we iterate here, we should probably create a field on Contacts module to track this ID used in Gradebook system.  I'll create a ticket.